### ISSUE TITLE
No file or directory

### ISSUE THREAD

**Issue by xz-123-new** [Arthur151/ROMP#494]
Thanks for your great work! I download the SMPL_python_v.1.1.0 according to the instruction in simple_romp, but no file called SMPL_NEUTRAL.pkl or SMPL_NEUTRAL.pth is available, would you please inform me where could i locate it or download it, thanks!

**Comment by Mayank-Maurya** [comment]
Hi, I think still there is no file, did you solved this issue, if yes, can you please explain, any help will be appreciated

**Comment by yayuanli** [comment]
Hi @Mayank-Maurya Mayank, I got the same issue here. Did you find the solution?

> Hi, I think still there is no file, did you solved this issue, if yes, can you please explain, any help will be appreciated

**Comment by yayuanli** [comment]
I think the basicmodel_neutral_lbs_10_207_0_v1.1.0.pkl in "Download version 1.1.0 for Python 2.7 (female/male/neutral, 300 shape PCs)" in [official website](https://smpl.is.tue.mpg.de/) is what they meant by SMPL_NEUTRAL.pkl

**Comment by Mayank-Maurya** [comment]
True, @yayuanli, SMPL is an easily available model that can be downloaded from multiple sources.
